pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -14,6 +14,9 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.time.Duration.ofSeconds;
```

Comment:
We've been inconsistent with this, but our preference has been to not import every static method.  In fact there are some ErrorProne inspections that will report as errors if we do . (*.Builder and *.of in particular).  So I think we should shy away from most method imports.

These particular imports causes a lot of changes in lines that are note relevant to the PR at hand.

Reply:
ok